assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/DraftIdeas.js

```diff
@@ -16,8 +16,109 @@
  * limitations under the License.
  */
 
-const DraftIdeas = () => {
-	return <div className="googlesitekit-idea-hub__draft-ideas" />;
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment, useCallback, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import {
+	IDEA_HUB_BUTTON_VIEW,
+	IDEA_HUB_IDEAS_PER_PAGE,
+	STORE_NAME,
+} from '../../../datastore/constants';
+import EmptyIcon from '../../../../../../svg/idea-hub-empty-draft-ideas.svg';
+import PreviewTable from '../../../../../components/PreviewTable';
+import Idea from './Idea';
+import Empty from './Empty';
+import Footer from './Footer';
+const { useSelect } = Data;
+
+const DraftIdeas = ( { active, WidgetReportError } ) => {
+	const [ page, setPage ] = useState( 1 );
+	const args = {
+		offset: ( ( page - 1 ) * IDEA_HUB_IDEAS_PER_PAGE ),
+		length: IDEA_HUB_IDEAS_PER_PAGE,
+	};
+	const totalDraftIdeas = useSelect( ( select ) => select( STORE_NAME ).getDraftPostIdeas()?.length );
+	const draftIdeas = useSelect( ( select ) => select( STORE_NAME ).getDraftPostIdeas( args ) );
+	const hasFinishedResolution = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getDraftPostIdeas', [ args ] ) );
+	const error = useSelect( ( select ) => select( STORE_NAME ).getErrorForSelector( 'getDraftPostIdeas', [ args ] ) );
+
+	const handlePrev = useCallback( () => {
+		if ( page > 1 ) {
+			setPage( page - 1 );
+		}
+	}, [ page, setPage ] );
+
+	const handleNext = useCallback( () => {
+		if ( page < Math.ceil( totalDraftIdeas / IDEA_HUB_IDEAS_PER_PAGE ) ) {
+			setPage( page + 1 );
+		}
+	}, [ page, setPage, totalDraftIdeas ] );
+
+	if ( ! active ) {
+		return null;
+	}
```

Comment:
>  Actually I was testing something and forgot to remove the active prop in index.js. I'am not sure though, that we need to actually return null here since the content is already hidden via CSS.

@asvinb let's remove it from here then and make sure we render only the active tab in the `index.js` file. No need to render all tabs and hide it with css.

Reply:
@eugene-manuilov If we render only the active tab then we'll lose the state, mostly for pagination when the tab is active again. That's why I opted for a CSS solution.

Reply:
Ah... ok. Thanks, @asvinb